ext/zip/tests/zip_entry_open.phpt

```diff
@@ -12,12 +12,12 @@ zip_close($zip);
 
 ?>
 --EXPECTF--
-Deprecated: Function zip_open() is deprecated in %s on line %d
+Deprecated: Function zip_open() is deprecated since 8.0, use ZipArchive::open() instead in %s on line %d
```

Comment:
Okay this is a complete aside, but I just realized that in the documentation, our manual style guide requires use to say "deprecated as of PHP 8.0" instead of "since", not sure if this is something we should match or not, but that's orthogonal to this PR.

Reply:
I think keeping consistency with the property name is valuable here.